docs/semgrep-sc/sc-getting-started.md

```diff
@@ -42,6 +45,8 @@ Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each
 This is the preferred method to enable and run Semgrep Supply Chain.
 :::
 
+Semgrep Supply Chain scans can be set up from the [Semgrep App](/) interface.
```

Comment:
@s-santillan just checking if this is not an omission, feel free to dismiss this comment. The `/` links to the hope page (resolve if this was the intention here). Maybe you wanted to link this to Semgrep App docs?

Reply:
Good catch, i'll fix it.